static_compress/mixin.py

```diff
@@ -1,11 +1,73 @@
+import os
+import errno
+from os.path import getatime, getctime, getmtime
+
+from django.core.exceptions import ImproperlyConfigured
+
 from . import compressors
 
 __all__ = ['CompressMixin']
 
 
+DEFAULT_METHODS = ['gz', 'br']
```

Comment:
The default should be `gz+zop` so it won't break existing users.

Plus, this module is all about trading build speed for filesize, so we should prefer the best options first.

Reply:
Hi, it won't break, because it still produce gzip files, just with bigger files.

Reply:
While it won't break apps, it would break the user's expectation